src/common/AndroidTypes.ts

```diff
@@ -17,14 +17,8 @@ export class AndroidPackages {
     /**
      * Initialize the logger used by AndroidPackages.
      */
-    public static async initializeLogger(
-        level?: LoggerLevelValue
-    ): Promise<void> {
```

Comment:
These were all remnant of old code. A long time ago we used to have each library create a logger (using `Logger.child`) whereas later on we moved away from that. Each library has a static instance of a logger and here we only need to set the level for the logger rather than create yet another child. I clean this up in all util classes.